CHANGELOG.md

```
@@ -2,7 +2,25 @@
 
 All notable changes to this repository.
 
-Current version: **2.4**
+Current version: **2.6**
+
+## [2.6] — 2026-04-19
+
+### Added
+- `scripts/lib/pr-body-parser.cjs` — utility for splitting/reassembling PR bodies on `## ` headers. Used by `/receiving-pr-feedback` to regenerate dynamic sections across feedback rounds.
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)